plugin/fs/plugin.go

```diff
@@ -19,7 +19,7 @@ func main() {
 		Version: "1.0.0",
 		Features: plugin.PluginFeatures{
 			Target: "yes",
-			Store:  "no",
+			Store:  "yes",
```

Comment:
We don't want to advertise that we do store ops.  Probably best to leave a comment that we don't advertise, but do implement the Retrieve endpoint, and why.